README.md

```diff
@@ -32,56 +31,82 @@ Please visit the [tutorial section](https://graspy.neurodata.io/tutorial.html) i
+- macOS x64
+- Windows 10 x64
+
+And across the following versions of Python:
+- 3.6 (x64)
+- 3.7 (x64)
+- 3.8 (x64)
 
-### Python Requirements
-This package is written for Python3. Currently, it is supported for Python 3.6 and 3.7.
+If you try to use `graspologic` for a different platform than the ones listed and notice any unexpected behavior,
+please feel free to [raise an issue](https://github.com/microsoft/graspologic/issues/new).  It's better for ourselves and our users 
+if we have concrete examples of things not working!
 
 ### Python Dependencies
-`GraSPy` mainly depends on the Python scientific stack.
+Users of `graspologic` will have the following direct dependencies:
 ```
+hyppo
+matplotlib
 networkx
 numpy
-pandas
+seaborn
 scikit-learn
 scipy
-seaborn
 ```
 
+Developers of `graspologic` will also have the following dependencies:
+```
+black
+ipykernel
+ipython
+myp
+nbsphinx
+numpydoc
+pandoc
+pytest
+pytest-cov
+sphinx
+sphinxcontrib-rawfiles
+sphinx-rtd-theme
+```
+
+Please note that `pandoc` will also [need to be installed for your system](https://pandoc.org/installing.html).
+
 # Installation Guide
 ## Install from pip
 ```
-pip install graspy
+pip install graspologic
 ```
 
 ## Install from Github
 ```
-git clone https://github.com/neurodata/graspy
-cd graspy
+git clone https://github.com/microsoft/graspologic
+cd graspologic
+python3 -m venv venv
+source venv/bin/activate
 python3 setup.py install
 ```
 
 # Contributing
-We welcome contributions from anyone. Please see our [contribution guidelines](https://graspy.neurodata.io/contributing.html) before making a pull request. Our 
-[issues](https://github.com/neurodata/graspy/issues) page is full of places we could use help! 
+We welcome contributions from anyone. Please see our [contribution guidelines](https://github.com/microsoft/graspologic/blob/dev/CONTRIBUTING.md) before making a pull request. Our 
+[issues](https://github.com/microsoft/graspologic/issues) page is full of places we could use help! 
 If you have an idea for an improvement not listed there, please 
-[make an issue](https://github.com/neurodata/graspy/issues/new) first so you can discuss with the 
-developers. 
+[make an issue](https://github.com/microsoft/graspologic/issues/new) first so you can discuss with the developers. 
 
 # License
 This project is covered under the MIT License.
 
 # Issues
-We appreciate detailed bug reports and feature requests (though we appreciate pull requests even more!). Please visit our [issues](https://github.com/neurodata/graspy/issues) page if you have questions or ideas.
+We appreciate detailed bug reports and feature requests (though we appreciate pull requests even more!). Please visit our [issues](https://github.com/microsoft/graspologic/issues) page if you have questions or ideas.
 
-# Citing GraSPy
-If you find GraSPy useful in your work, please cite the package via the [GraSPy paper](http://www.jmlr.org/papers/volume20/19-490/19-490.pdf)
+# Citing `graspologic`
```

Comment:
Am I using the right words for this?  Should we be Citing `GraSPy`, but leave `graspologic` where it is down below?  Plz to check me

Reply:
this seems fine to me for now... years from now i could imagine it would be confusing to people but I'm not sure what to do about it.